tests/Makefile

```diff
@@ -44,7 +44,7 @@ ifdef OpenCL_INCPATH
 	CXXFLAGS += -I$(OpenCL_INCPATH)
 endif
 
-# Specify where Bost.Compute is with BOOST_COMPUTE_INCPATH
+# Specify where Boost.Compute is with BOOST_COMPUTE_INCPATH
```

Comment:
This commit is already in master as "fix a comment (#5)" https://github.com/keryell/triSYCL/commit/0f5a041f44bdfa6bd41d39acfc800d448b4a2525
So I have the feeling this commit should not be here.